controllers/controllers.js

```diff
@@ -30,3 +30,14 @@ exports.getReviewById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  selectCommentsByReviewId(review_id)
+    .then((commentsByReviewId) => {
+      res.status(200).send({ commentsByReviewId });
```

Comment:
I would recommend renaming your `commentsByReviewId` variable to just `comments` - it's a bit more simple and user friendly for developers who are using your API - which will be yourself when you come to the front end project

Just remember that if you change the name here then you must also change the name of it in your test suite when you run the tests for the endpoint so they are looking for the correct property on the `body`